component/src/main/java/org/wso2/rule/validator/validator/JsonRulesetValidator.java

```diff
@@ -28,7 +29,12 @@
  * Ruleset validation for JSON files
  */
 public class JsonRulesetValidator extends RulesetValidator {
-    public static List<RulesetValidationError> validateRuleset(String rulesetString) {
-        return RulesetValidator.validate((Map<String, Object>) JsonPath.parse(rulesetString).json());
+    public static List<RulesetValidationError> validateRuleset(String rulesetString)
+            throws InvalidContentTypeException {
+        Object jsonContent = JsonPath.parse(rulesetString).json();
```

Comment:
Could the `.json()` call throw a `NullPointerException` if we provide an empty string?

Reply:
If the string is empty this file won't be reached right? If that's the case we don't need to fix it.